simple-calculator/index.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title>
+		My Calculator
+	</title>
+	<style type="text/css">
+		body
+		{
+			background-color:#F5DEB3;
+		}
+		form
+		{
+			
+			
+			margin-top: 100px;
+			margin-left: 1000px;
+			
+		}
+		.pi
+		{
+			height:60px;
+			text-align: center;
+			background-color: #FFF8DC;
+			border-style: none;
+			
+		}
+		.yi
+		{
+			border-radius: 25px;
+			background-color: #FFFFE0;
+			
+		}
+		div{
+			font-family:Courier New;
+			font-size: 30pt;
+
+		}
+	</style>
+</head>
+<script src="mycalculator.js"></script>
+<body>
+<div><br> 
+ 
+  TRY MY CALCULATOR HUMAN !
+  <br>IT IS FASTER THAN THE FASTEST OF ALL OF YOU .<div>
+	<form name="fo">
```

Comment:
> **Practicality over purity**
> Strive to maintain HTML standards and semantics, but not at the expense of practicality. Use the least amount of markup with the fewest intricacies whenever possible. 

The form is not needed here I guess, you can remove it.
and maybe wrap your titles in `<h2>` and `<h4>`